packages/theme/src/utils/cookies.ts

```diff
@@ -0,0 +1,12 @@
+import { ThemeName } from '@lidofinance/theme'
+import { themeCookieKey } from '../constants'
+
+const cookieThemeMatcher = new RegExp(`(^| )${themeCookieKey}=([^;]+)`)
```

Comment:
Maybe we should mention that we have similar regex in [element-theme-script.tsx](https://github.com/lidofinance/ui/pull/350/files#diff-e8dbc3eda19e57ac79a8698d709af83bea63dd9d3071cedf2b661533a94bce72)?